src/main/kotlin/io/github/nomisrev/routes/articles.kt

```diff
@@ -41,6 +71,35 @@ data class Comment(
   val author: Profile
 )
 
+@Resource("/article")
+data class ArticleResource(val parent: RootResource = RootResource) {
+  @Resource("/feed") data class Feed(val parent: ArticleResource = ArticleResource())
+}
+
+fun Route.articleRoutes(
+  articleService: ArticleService,
+  jwtService: JwtService,
+) {
+
+  get<ArticleResource.Feed> {
```

Comment:
We have access to `ArticleResource.Feed` here, and thus we get access to `offsetParam` and `limitParam`.

```suggestion
  get<ArticleResource.Feed> { feed ->
```